src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -769,7 +785,7 @@ public PreferencesViewModel(DynamoViewModel dynamoViewModel)
             StyleItemsList = LoadStyles(preferenceSettings.GroupStyleItemsList);
           
             //When pressing the "Add Style" button some controls will be shown with some values by default so later they can be populated by the user
-            AddStyleControl = new StyleItem() { GroupName = "", HexColorString = "#" + GetRandomHexStringColor() };
+            AddStyleControl = new StyleItem() { Name = "", HexColorString = GetRandomHexStringColor() };
```

Comment:
Please use string.empty where it applies, thanks!

Reply:
@QilongTang fixed in the next commit: https://github.com/DynamoDS/Dynamo/pull/12667/commits/5fc4b1e35ba1ad5b280d4b412fa73f762bed056e